### DISCUSSION TITLE
Launching the App

### DISCUSSION THREAD

**Discussion by CartwrM** [pharmaR/risk_assessment#287] [Q&A] [answered]
Hi,
   I was previously able to launch the app using
`runUrl("https://github.com/pharmaR/risk_assessment/archive/master.zip",
       subdir = "inst/app/")`

But this is now failing with the error: "App dir must contain either app.R or server.R."

I've also tried:
`shiny::runGitHub('risk_assessment', 'pharmaR')`

and 
`shiny::runGitHub('risk_assessment', 'pharmaR', ref="main")`

to no avail.

Please help!

Thanks

**Comment by AARON-CLARK** [comment, marked as answer]
@CartwrM, thanks for bringing this up and please feel free to open a GitHub issue next time!

There have been some major changes on the repo recently that leveraged `{golem}` to convert the app to a package. As such, a traditional `app.R` file wasn't/isn't needed until you are ready to deploy the app. However, we knew we'd need one eventually to deploy on shinyapps.io, so we just added one to the root directory of the package in response to your 
 question. That is, you should now be able to run either of the following to run the app: 
```
shiny::runGitHub('risk_assessment', 'pharmaR')
shiny::runUrl("https://github.com/pharmaR/risk_assessment/archive/master.zip")
```
We are actively developing new features on the `dev` branch and will periodically push fresh code to `master` so stay tuned for more upcoming enhancements!
